POS/Overview/02_Features.md

```diff
@@ -1,4 +1,5 @@
 # Key features
+<!---Error-->
 
```

Comment:
Vorchläge:
- The *Venduo POS* module of Actindo offers a complete checkout system with all necessary functions, such as flexible payment methods, returns management or voucher handling -> The Actindo *Venduo POS* module offers a complete checkout system with all necessary functions, such as flexible payment methods, returns management, or voucher handling.
- You can easily connect pop-up stores. The assortment for each store can then be controlled from our *PIM* module. You can assign a complete user history, such as log-in and log-out data, sales data, and stocks to the individual POS systems as well as to users. --> Was sind "pop-up stores"? Der Term kommt mir neu vor. Ist mit "assortment" "stock value" gemeint? Der Rest des Satzes kommt mir auch unbekannt vor, aber vielleicht liegt das an mir. Oder hat das mit der Profil-Funtion zu tun? Die ist noch nicht released, oder?
- With automatic data storage and retention, the *Venduo POS* module generates your sales, stuff... -> Vermtutlich "staff" gemeint?
- Orders and returns from stores are thus displayed together with your online sales channels. --> Ich weiß nicht, ob das so deutlich ist. POS-Bestellungen werden wie alle andere Bestellungen (aus sales channels) behandelt, also via OC ins System (POS ist eine OC-Verbindung), dann auch in OMS und Accounting, ggf. RMA, weiter bearbeitet. Meinst du das?
- (Kassensicherungsverordnung (KassenSichV Germany) -> Vorschlag: ... according to / in compliance with German legal requirements (Kassensicherungsverordnung). (Oder so etwas). Auf jeden Fall gibt es ein Klammer zu viel oder zu wenig jetzt.